e2e/tests/app.e2e.js

```diff
@@ -0,0 +1,44 @@
+import { expect, test } from '@playwright/test';
+import { NavBarPage } from '../pages/nav-bar-page';
+import { HomePage } from '../pages/home-page';
+
+test.describe('AngularJS-Material-Vite app', () => {
```

Comment:
As with other repos, I think it is valuable to assert that there are no thrown/logged errors in the console after each test.

Reply:
Changed